### DISCUSSION TITLE
Noticed the settings in "Smoothfox.js" is not included in "User.js

### DISCUSSION THREAD

**Discussion by Feejus** [yokoffing/Betterfox#283] [General]
I could not find any documentation on this, but I had the understanding that the user.js file would include all other settings by default. However, I noticed that the settings from Smoothfox.js are not included in my user.js file.

Is my understanding correct that to apply the settings from Smoothfox.js, I need to manually copy those preferences and add them to my user.js file?

Thanx!

**Comment by yokoffing** [comment]
> Is my understanding correct that to apply the settings from Smoothfox.js, I need to manually copy those preferences and add them to my user.js file?

Yes.
1. https://github.com/yokoffing/Betterfox/blob/c36643914571d10b45f863e3916441071b838ae3/user.js#L238-L242
2. https://github.com/yokoffing/Betterfox/wiki/Common-Overrides#instructions > https://github.com/yokoffing/Betterfox/wiki/Common-Overrides#scrolling
